ansible/roles/edu-sharing/tasks/edu-sharing-docker/main.yml

```diff
@@ -0,0 +1,91 @@
+---
+- name: Load version variables.
+  with_first_found:
+    - '../vars/versions/{{ edu_version }}.yml'
+    - '../vars/versions/default.yml'
+  include_vars: '{{ item }}'
+
+- name: Check if Migration has been installed before
+  shell: "echo ${EDU_SHARING_DOCKER_MIGRATION:-false}"
```

Comment:
Do we really need an environment variable to detect this? Isn't it just possible to verify if a new installation already exists? If so, regarding the naming `EDU_SHARING_DOCKER_MIGRATION` - this could mean "Do the migration" or this could mean "Migration was procssed" - so not clear